easy_my_coop/view/subscription_template.xml

```diff
@@ -80,27 +80,34 @@
 			           <div name="email_from_container" t-attf-class="form-group #{error and 'email_from' in error and 'has-error' or ''}">
 			                <label class="col-md-3 col-sm-4 control-label" for="email">Email</label>
 			                <div class="col-md-7 col-sm-8 bottom-line" style="padding-bottom:20px">
-			                    <input type="email" class="form-control mandatory-field" name="email" required="True" t-attf-value="#{email or ''}" placeholder="didier.bourdon@bees-coop.be"/>
+			                    <input type="email" class="form-control mandatory-field" name="email" 
+			                    required="True" t-att-readonly="logged"
```

Comment:
aligned attributes would please my eyes <3

Reply:
indeed ! required alone lead to lxml error so I've replaced it by required="required"
In fact t-att-required="True" should work but it should be used with a boolean variable as it's done for readonly attribute.